src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.Designer.cs

```diff
@@ -43,17 +43,24 @@ private void InitializeComponent()
             this.NewPackageSource = new System.Windows.Forms.TextBox();
             this.NewPackageNameLabel = new System.Windows.Forms.Label();
             this.NewPackageName = new System.Windows.Forms.TextBox();
-            this.PackageSourcesListBox = new NuGet.Options.PackageSourceCheckedListBox();
             this.tableLayoutPanel1 = new System.Windows.Forms.TableLayoutPanel();
             this.tableLayoutPanel2 = new System.Windows.Forms.TableLayoutPanel();
             this.addButton = new System.Windows.Forms.Button();
+            this.PackageSourcesListBox = new NuGet.Options.PackageSourceCheckedListBox();
             this.MachineWideSourcesLabel = new System.Windows.Forms.Label();
             this.MachineWidePackageSourcesListBox = new NuGet.Options.PackageSourceCheckedListBox();
+            this.tableLayoutPanel3 = new System.Windows.Forms.TableLayoutPanel();
+            this.tableLayoutPanel4 = new System.Windows.Forms.TableLayoutPanel();
```

Comment:
I think this panel is a bit too small, or is being affected by another margin/etc.
My options dialog is as small as it goes, and it chops off the last word of the message.
![image](https://user-images.githubusercontent.com/49205731/174390245-633390ba-ea44-4d23-86cb-bf931fa81b8f.png)

Reply:
@donnie-msft I moved the label to a new row, I think this fixes the problem and look better

![image](https://user-images.githubusercontent.com/43253759/174636604-636c30b0-4129-444d-87fd-d7b6703ebe52.png)
![image](https://user-images.githubusercontent.com/43253759/174636704-068c6b35-02f9-400a-937b-8bb13dbf638d.png)

Reply:
The minimum width issue is resolved.